algorithms/CSharp/src/Maths/abundant-number.cs

```diff
@@ -0,0 +1,48 @@
+/* Program to check whether a number is abundant or not */
+using System;
+
+namespace abundant_number
+{
+    class Solution
+    { 
+        // function to check whether number is abundant or not
+        public static bool Abudant(int num)
```

Comment:
Rename the method name to `IsAbudant`
```suggestion
        public static bool IsAbudant(int num)
```